spec/authentication.yaml

```
@@ -167,9 +167,8 @@ components:
           in: "query"
           description: "(**Optional**) role specifier in `{kind}:{identifier}` format
 
-            openapi.yml#openapi.yml#openapi.yml#openapi.yml#openapi.yml# Permissions required
-
-            `update` privilege on the role whose API key is being rotated."
+            Permissions required:
```

Other spec files use this format for headers, came across this in `spec/roles.yml` as well:
```suggestion
            ##### Permissions required:

```